js_modules/dagit/packages/core/src/runs/RunsFilterInput.tsx

```diff
@@ -21,6 +22,11 @@ type PipelineRunTags = RunsSearchSpaceQuery_pipelineRunTags[];
 
 export type RunFilterTokenType = 'id' | 'status' | 'pipeline' | 'job' | 'snapshotId' | 'tag';
 
+export type RunFilterToken = {
```

Comment:
Is there an advantage to this and having to do a token === {expectedType} vs just doing a traditional object as 

`{id?:string, status?:string; pipeline?:string; job?:string; snapshotId?: string; tag?:string}`

Reply:
ah, i think i get it. compatibility with a bunch of stuff thats already in Tokenizing Field, which has the possibility of no token at all...

Reply:
Ahh yeah, I think the token key/value shape just makes this more compatible with the "tokenizing" component 👍